tools/semaphore-run

```diff
@@ -3,6 +3,12 @@
 set -o pipefail
 set -eux
 
+# on semaphore the old npm in ~/.nvm is preferred over /usr/local, where "g stable" ends up
+if hostname | grep -q ^semaphore; then
```

Comment:
do we ever use this outside of semaphore?

Reply:
I often run these in am Ubuntu 14.04 chroot to reproduce semaphore failures.